twisted-infra/braid
# Move buildot config in braid and fix testing buildmaster.

Problem
=======

testing data is broken as slaves are missing.

I tried to fix this while used the Vagrant VM but find it horrible to work with  current submodules structure... I have already mentioned this problem here https://github.com/twisted-infra/braid/issues/79

Changes
========

this is based/depends on #77

convert services/buildbot submodule into a simple folder.

Manual update of private.sample.

Update the environment to work with vagrant

The diff is big due to the submodule convertion but I kept them in a separate commit.

You can see branch specific changes here https://github.com/chevah/braid/commit/b6cc91ac5b0a32e1d965b1630e00b8d0670810e1

How to test
==========

Once you have the vagrant VM up and running run

```
fab -i ~/.vagrant.d/insecure_private_key config.vagrant buildbot.install
fab -i ~/.vagrant.d/insecure_private_key config.vagrant buildbot.start
fab -i ~/.vagrant.d/insecure_private_key config.vagrant buildbot.log
```

Check buildmaster at http://172.16.255.140:8080/
